pkgs/tools/networking/dnscrypt-proxy/2.x/default.nix

```diff
@@ -1,24 +1,19 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, buildGoPackage, fetchzip }:
 
 buildGoPackage rec {
-  name = "dnscrypt-proxy2-${version}";
-  version = "2.0.15";
+  name = "dnscrypt-proxy-${version}";
+  version = "2.0.17";
 
   goPackagePath = "github.com/jedisct1/dnscrypt-proxy";
 
-  src = fetchFromGitHub {
-    owner = "jedisct1";
-    repo = "dnscrypt-proxy";
-    rev = "${version}";
-    sha256 = "0iwvndk1h550zmwhwablb0smv9n2l51hqbmzj354mcgi6frnx819";
+  src = fetchzip {
+    url = "https://${goPackagePath}/archive/${version}.tar.gz";
+    sha256 = "12mg1jbla2hyq44wq9009sqb0mzwq16wi8shafabwk0zf9s2944d";
   };
 
   meta = with stdenv.lib; {
-    description = "A tool that provides secure DNS resolution";
-
+    description = "DNS over HTTPS and DNSCrypt proxy";
     license = licenses.isc;
-    homepage = https://dnscrypt.info/;
```

Comment:
Why remove `homepage`?

Reply:
This is not the actual project homepage. `buildGoPackage` generates correct `meta.homepage` for this package.